bigbluebutton-html5/imports/ui/components/layout/layout-manager/container.jsx

```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+import { withTracker } from 'meteor/react-meteor-data';
+import Auth from '/imports/ui/services/auth';
+import Screenshare from '/imports/api/screenshare';
+import LayoutManagerComponent from '/imports/ui/components/layout/layout-manager/component';
+
+const LayoutManagerContainer = ({ screenIsShared }) => (
+  <LayoutManagerComponent {...{ screenIsShared }} />
+);
+
+export default withTracker(() => {
+  const screenshareEntry = Screenshare.findOne({ meetingId: Auth.meetingID },
```

Comment:
There's an identical piece of code in: https://github.com/bigbluebutton/bigbluebutton/blob/7fa9f97c74d1eca2cdb20873b03e48155afcc128/bigbluebutton-html5/imports/ui/components/screenshare/service.js#L33-L41

Can you please re-use it?